org.mwc.debrief.lite/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -1,5 +1,6 @@
 eclipse.preferences.version=1
 org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.methodParameters=do not generate

```

Comment:
@ayeshare - for my self-teaching, could you explain why we've changed this setting?

Reply:
Aah, I see. I guess Eclipse just decided to insert it.